RELEASE_PROCESS.md

```diff
@@ -25,6 +25,13 @@ development efforts occur on the *main* branch.  Branches with a
 * You will announce the release on the proper platforms
   (i.e. Podman blog, Twitter, Mastodon Podman and Podman-Desktop mailing lists)
 
+# Prechecks
+
+Two days before actually cutting the release (including RCs), send an announcement on the
+[podman](mailto:podman@lists.podman.io) and [podman-desktop](mailto:podman-desktop@lists.podman.io)
+mailing lists about the upcoming release. This allows sufficient time to check if any open
+features / fixes need to be included in the release.
```

Comment:
I don't understand the context behind this? We never did something like this in the past AFAICT. What is the reason behind this? Release blockers should always be communicated ASAP and not just two days before.
IMO two days is way to little time if there is any actual work to do, i.e debug/fix/test critical bug, etc...

To be clear of I have no problem with doing this but I say we should communicate this much early to reduce any last minute stress.

Reply:
well, i assumed the context. @benoitf any reason for 1-2 days you mentioned?

Reply:
I think it's more to check the bits using Podman Desktop

But it's also to know when a release is happening as we need to schedule a release on our side as well. It may also avoid issues where we cut a release of Podman Desktop and only few hours after a bugfix release of Podman occurred.

Reply:
great. I can update the doc with this. thanks @benoitf

Reply:
I guess an advance announcement on podman-desktop would suffice in that case. I'll remove advance announcement on the podman mailing list.

Reply:
1-2 day was mostly to know when release is happening and not discover it 2 days after it is tagged.

Reply:
updated. PTAL

Reply:
I think there is nothing wrong with announcing it on both lists but I would say the earlier the better, internally we know when the release happens at least one week before. I see no problem with announcing that publicly when we agreed to a date. That would at signal community contributors to finish there work by then if they want it to be included in that release.

Reply:
if you know one week before, yes just announce it and do not wait 1-2 days before to announce it publicly ! :-)

the goal is to have an early notification and not a post-notification